inclusive-design/wecount.inclusivedesign.ca
# feat: show admin link for logged in administrators (resolves #475)

## Description

This PR uses the Netlify Identity widget's `login` hook to show an administration link to logged-in administrators.

## Steps to test

1. Log in as an administrator from the home page.
2. Log out.
3. Log in as a non-administrator from the home page.

**Expected behavior:** After step 1, an "Administration" link appears above the "Logout" link. After step 3, no "Administration" link appears.

## Additional information

- https://github.com/netlify/netlify-identity-widget#usage

## Related issues

Resolves #475.
